docs/docs/Agents/agents.md

```diff
@@ -0,0 +1,232 @@
+The Playground displays the agent's tool calls, what input was provided, and the raw output the agent received before performing the summary. The agent should call the News Search component's `search_news` action.
+
+You've successfully constructed a flow with the Langflow Agent.
+Connect more tools to solve more specialized problems.
+
+### Edit a tool component's actions
+
+To edit a tool's actions, in the tool component, click <Icon name="SlidersHorizontal" aria-hidden="true"/> **Edit Tools** to modify its `name`, `description`, or `enabled` metadata.
+These fields help connected agents understand how to use the action, without having to modify the agent's prompt instructions.
+
+For example, the [URL](/components-data#url) component has two actions available when **Tool Mode** is enabled.
+
+| Tool Name | Description | Enabled |
+|-----------|-------------|---------|
+| `fetch_content` | Fetch content from web pages recursively | true |
+| `fetch_content_as_message` | Fetch web content formatted as messages | true |
+
+With these descriptions, a connected agent has a clear idea of each tool's capabilities based on the `name` and `description` metadata. The `enabled` boolean controls the tool's availability to the agent. If you think an agent is using a tool incorrectly, edit a tool's `description` metadata to help the agent better understand the tool.
+
+Tool names and descriptions can be edited, but the default tool identifiers cannot be changed. If you want to change the tool identifier, create a custom component.
+
+## Use an agent as a tool
+
+The agent component itself also supports **Tool Mode** for creating multi-agent flows.
+
+Add an agent to your flow that uses a different OpenAI model for a larger context window.
+
+1. Click and drag an **Agent** component to your workspace.
+2. Add your **Open AI API Key** to the **Agent** component.
+3. In the **Model Name** field, select `gpt-4.1`.
+4. Click **Tool Mode** to use this new agent as a tool.
+5. Connect the new agent's **Toolset** port to the previously created agent's **Tools** port.
+6. Connect the **Web Search**, **URL**, and **Calculator** to the new agent.
+The new agent will use `gpt-4.1` for the larger tasks of scraping and searching information that require large context windows.
+The previously created agent will now use this agent as a tool, with its unique LLM and toolset.
+
+![Agent as a tool](/img/agent-example-agent-as-tool.png)
+
+7. The new agent's actions can be edited to help the agent understand how to use it.
+Click <Icon name="SlidersHorizontal" aria-hidden="true"/> **Edit Tools** to modify its `name`, `description`, or `enabled` metadata.
+For example, the default tool name is `Agent`. Edit the name to `Agent-gpt-41`, and edit the description to `Use the gpt-4.1 model for complex problem solving`. The connected agent will understand that this is the `gpt-4.1` agent, and will use it for tasks requiring a larger context window.
```

Comment:
@edwinjosechittilappilly This is the bug that I reported, where the connected agent as tool displays potentially incorrect information.